remixer_storage/build.gradle

```diff
@@ -16,6 +16,7 @@
 
 apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'jacoco-android'
```

Comment:
are you sure this works? How do you know? I don't see anything on https://codecov.io/gh/material-foundation/material-remixer-android

See my description of http://codereview.cc/D1777